jormungandr-integration-tests/src/common/jcli_wrapper/mod.rs

```diff
@@ -398,3 +398,10 @@ pub fn assert_get_rest_message_log(host: &str) -> Vec<FragmentLog> {
         serde_yaml::from_str(&content).expect("Failed to parse fragment log");
     fragments
 }
+
+pub fn assert_shutdown_node(host: &str) {
+    let output = process_utils::run_process_and_get_output(
+        jcli_commands::get_rest_shutdown_node_command(&host),
+    );
+    process_assert::assert_process_exited_successfully(output);
```

Comment:
This will be failing randomly yet constantly. When REST shutdown endpoint responds, it means that the shutdown process has started, not ended!

Reply:
So basically to be sure node is not up. i need to send some message to it (like node get stats) and assure i got no connection error or this information could be retrieved from logs ?

Reply:
Sorry, I've missed the point of this function. Assertion checks if JCLI command exited successfully, not node itself. Probably failures will be more subtle then, because if the node still runs for a moment after that, we won't get a direct panic from here.

I think that the perfect solution would be to check if the process crated on node start is down, that gives 100% reliability.

Looking for `info` `"Shutting down node"` in logs is almost as good, the node usually will be running after logging that just for a fraction of a second. The logs could be even flushed during process cleanup by OS, when node is already down. But it's still relying on something random.